README.md

```diff
@@ -92,6 +92,18 @@ If you wish to see another distribution on the list, please test on the latest c
 Since version `1.1.0` we no longer default to using `main` but use tags to make sure deployments are versioned.
 With every new release all migration steps shall be written below so make sure you check out the [Lemmy Releases Changelog](https://github.com/LemmyNet/lemmy/blob/main/RELEASES.md) to see if there are any config changes with the releases since your last read.
 
+### Upgrading to 1.4.0 (Lemmy 0.19.3 & Pict-rs 0.5.4)
+
+This is a semi-major release which upgrades pict-rs to 0.5 which has support for postgres as a backend. This configuration is **not supported** by lemmy-ansible for the moment.
+
+#### Steps
+
+- `git pull && git checkout 1.4.0`
+- Read [Pictrs' Configuration Changes](https://git.asonix.dog/asonix/pict-rs/#configuration-updates)
+- Ammend your `vars.yml` file to respect the new changes
+  - Optional: Add: `PICTRS__UPGRADE__CONCURRENCY` with a value between 32 and 512 depending on how much RAM/CPU you want to dedicate to the upgrade process. A value of 32 used about 2.5GB of RAM for the migration.
+- Run your regular deployment. Example: `ansible-playbook -i inventory/hosts lemmy.yml --become`
```

Comment:
Also mention about running `POST /internal/prepare_upgrade` with 0.4 to reduce downtime with the migration. Its documented [here](https://git.asonix.dog/asonix/pict-rs/src/branch/v0.4.x), Ctrl+F for `prepare_upgrade`.

Reply:
It was added as an optional note but the conversation didn't get marked as resolved. I think this should be sufficient, unless you're looking for a more prominent mention for larger instances.

https://github.com/LemmyNet/lemmy-ansible/pull/213/commits/1c33460ffcd6f88191692ce0ccec057f96712a0a